src/main/kotlin/no/nav/syfo/model/PdfModel.kt

```diff
@@ -8,3 +13,23 @@ data class PdfModel(
     val navnSignerendeLege: String,
     val antallVedlegg: Int,
 )
+
+const val BYTE_ORDER_MARK = '\uFEFF'
+const val INVALID_CHAR = '\uFFFE'
+const val NUL = 0.toChar()
+
+fun PdfModel.sanitizeForPdfGen(): PdfModel {
+    val pdfModelJsonString = objectMapper.writeValueAsString(this)
+
+    val sanitizedJson = pdfModelJsonString.toCharArray().filter {
+        if (it > NUL && it != BYTE_ORDER_MARK && it != INVALID_CHAR) {
```

Comment:
Kunne det vært en idé og lagt de ulovlige tegnene i en liste og sjekket `it not in illegalList`? Det kommer kanskje "aldri" til å bli flere ulovlige tegn, så da er det kanskje unødvendig og mindre tydelig med liste?